proto/prysm/v1alpha1/attestation.go

```diff
@@ -156,6 +167,14 @@ func (a *PendingAttestation) GetSignature() []byte {
 	return nil
 }
 
+// GetCommitteeIndex --
+func (a *PendingAttestation) GetCommitteeIndex() (primitives.CommitteeIndex, error) {
```

Comment:
Why is there no PendingAttestationElectra needed? wouldn't it be returning the wrong committee index in electra?